contracts/EthStorageContract.sol

```diff
@@ -2,27 +2,47 @@
 pragma solidity ^0.8.0;
 
 import "./StorageContract.sol";
-import "./Decoder.sol";
-import "./BinaryRelated.sol";
+import "./zk-verify/Decoder.sol";
+import "./libraries//BinaryRelated.sol";
 
+/// @custom:proxied
+/// @title EthStorageContract
+/// @notice EthStorage Contract that using EIP-4844 BLOB
 contract EthStorageContract is StorageContract, Decoder {
-    uint256 constant modulusBls = 0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001;
-    uint256 constant ruBls = 0x564c0a11a0f704f4fc3e8acfe0f8245f0ad1347b378fbf96e206da11a5d36306;
-    uint256 constant ruBn254 = 0x931d596de2fd10f01ddd073fd5a90a976f169c76f039bb91c4775720042d43a;
-    uint256 constant modulusBn254 = 0x30644e72e131a029b85045b68181585d2833e84879b9709143e1f593f0000001;
-    uint256 constant fieldElementsPerBlob = 0x1000;
+    /// @notice The modulus for the BLS curve
+    uint256 internal constant MODULUS_BLS = 0x73eda753299d7d483339d80809a1d80553bda402fffe5bfeffffffff00000001;
+
+    /// @notice The root of unity of BLS curve
+    uint256 internal constant RU_BLS = 0x564c0a11a0f704f4fc3e8acfe0f8245f0ad1347b378fbf96e206da11a5d36306;
+
+    /// @notice The root of unity for the BN254 curve
+    uint256 constant RU_BN254 = 0x931d596de2fd10f01ddd073fd5a90a976f169c76f039bb91c4775720042d43a;
+
+    /// @notice The modulus for the BN254 curve
+    uint256 constant MODULUS_BN254 = 0x30644e72e131a029b85045b68181585d2833e84879b9709143e1f593f0000001;
+
+    /// @notice The field elements per BLOB
+    uint256 constant FIELD_ELEMENTS_PER_BLOB = 0x1000;
 
     // TODO: Reserve extra slots (to a total of 50?) in the storage layout for future upgrades
 
+    /// @notice Emitted when a BLOB is appended.
+    /// @param kvIdx    The index of the KV pair
+    /// @param kvSize   The size of the KV pair
+    /// @param dataHash The hash of the data
     event PutBlob(uint256 indexed kvIdx, uint256 indexed kvSize, bytes32 indexed dataHash);
 
+    /// @notice Constructs the EthStorageContract contract.
     constructor(
         Config memory _config,
         uint256 _startTime,
         uint256 _storageCost,
         uint256 _dcfFactor
-    ) StorageContract(_config, _startTime, _storageCost, _dcfFactor) {}
+    ) StorageContract(_config, _startTime, _storageCost, _dcfFactor) {
+        //initialize(0, 0, 0, address(0x0), msg.sender);
```

Comment:
Should it be deleted? The constructor should not call this function.

Reply:
You can check the guide here: https://github.com/ethereum-optimism/optimism/blob/457f33f4fdda9373dcf2839619ebf67182ee5057/packages/contracts-bedrock/STYLE_GUIDE.md?plain=1#L109